analyzers/tests/SonarAnalyzer.UnitTest/TestCases/PreferGuidEmpty.CSharp9.cs

```diff
@@ -16,11 +16,6 @@ public void Foo()
             string test = default;
         }
 
-        // See: https://github.com/SonarSource/sonar-dotnet/issues/5245
-        private void Test(Guid guid = default) // Noncompliant FP - Guid.Empty cannot be used in this case
-        {
-        }
-
```

Comment:
Why was this removed? I expected it to be fixed by this PR, or left as FP (and we'll update the description that this one was not fixed)

Reply:
It is on the not specific C# 9 version (and supported now). Felt an obsolete extra test.